sdk/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "basyx-python-framework"
+version = "0.1"
+dependencies = ["aas-core3.0"]
+requires-python = ">=3.8"
```

Comment:
Can you double check which versions of Python the BaSyx-Python-SDK supports (in the `.github/workflow/ci.yaml`) and adapt this here, if necessary?

Reply:
Okay, in the `.github/workflow/ci.yaml` the test job (which I guess would be the only one actually interacting with the code) uses python `["3.8", "3.10", "3.12"]`.
So I think those would be the supported versions?
We might have to change the pyproject.toml if we aim to match the test job specification.
E.g.
`requires-python = "==3.8 || ==3.10 || ==3.12"` (Not so sure about the syntax)

Reply:
We assume that the in-between versions would also work, even though we're only testing those exact three versions. 
Therefore, I'd write here "between 3.8 and 3.12", however the syntax for that is.

Reply:
Okay I looked it up and adjusted the specification to:
`requires-python = ">=3.8, <=3.12"`
which should be valid TOML syntax.